### PR TITLE
Feature/statistics/5.x

### PR DESCRIPTION
**Pull request by lestarcdog**
Please review if if you have time. Most of the things are PRed to the iceland branch also they should be functioning together.

### CODE REVIEW

File: bindings/rest/xml/pom.xml [hunk 1]
```diff
@@ -55,5 +55,34 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
             </plugin>
         </plugins>
+        <pluginManagement>
```
Comment (nuest) [inline bindings/rest/xml/pom.xml]: @autermann is this allowed? :-)

Comment (autermann) [inline bindings/rest/xml/pom.xml]: You mean a `<plugins>` and a `<pluginManagement>` section? Of course it is...

Comment (nuest) [inline bindings/rest/xml/pom.xml]: I mean adding a lifecycle mapping for Eclipse...

Comment (lestarcdog) [inline bindings/rest/xml/pom.xml]: This lifecycle mapping appears in other places also.

**Pull request closed without merging**